src/Event/Productions/Suggestion.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Productions;
+
+class Suggestion
```

Comment:
Can't we re-use the existing `SimilarEventPair` for this?

Reply:
Technically we could, but semantically they are two different things. They happen to have the same technical implementation but they are two different things so I would not opt using the SimilarEventPair here to avoid confusion.